build/templates/olm-artifacts-template.yaml.tmpl

```diff
@@ -63,3 +63,16 @@ objects:
         name: configure-alertmanager-operator
         source: configure-alertmanager-operator-registry
         sourceNamespace: openshift-monitoring
+    - apiVersion: rbac.authorization.k8s.io/v1
+      kind: ClusterRoleBinding
+      metadata:
+        name: configure-alertmanager-operator.prom
+        namespace: openshift-monitoring
```

Comment:
CRBs are cluster scoped. This can be dropped

Reply:
Oh. Yeah.

Reply:
Done.